src/anm/animation/animation.js

```diff
@@ -166,11 +167,34 @@ Animation.prototype.traverse = function(visitor, data) {
  *
  * @param {Function} visitor
  * @param {anm.Element} visitor.child
+ * @param {Boolean} visitor.return if `false` returned, stops the iteration. no-`return` or empty `return` both considered `true`.
  * @param {Object} [data]
  */
 Animation.prototype.each = function(visitor, data) {
     for (var i = 0, tlen = this.tree.length; i < tlen; i++) {
-        visitor(this.tree[i], data);
+        if (visitor(this.tree[i], data) === false) break;
```

Comment:
for JavaScript it's a bit different since if you omit `return` (since it's quite easy to forget it), it's commonly considered _truthy_, and `false` value may only be achieved with explicitly returning `false`. It's only regarding the return values from functions. At least it was for `jQuery.each` and it's there for different implementations.

Though, in modern Javascript there's no way to break out of `Array.each` (probably for the sake of clean code, indeed), but there's a special [`Array.every`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) method which works differently.

@mcm69 what would you prefer in this case?